crispycuts/src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
```

Comment:
Delete line 9

Reply:
Actually, delete all the html in App except for the outer div and inside the div add you main component so it renders on launch.

Reply:
You'll need to import the component though.

Reply:
I deleted all the html, still need to import the component